apps/charterafrica/src/lib/ecosystem/github/processData.js

```diff
@@ -47,23 +47,56 @@ const GET_REPOSITORY = `query($repositoryOwner: String!, $repositoryName: String
     }
   }`;
 
-async function fetchRepository(variables) {
+const GET_USER = `query($username: String!) {
+  user(login: $username) {
+    name
+    avatarUrl
+    url
+    email
+    location
+    login
+    websiteUrl
+    repositories(first: 3, orderBy: {field: STARGAZERS, direction: DESC}) {
+      edges {
+        node {
+          name
+          description
+          stargazers {
+            totalCount
+          }
+          visibility
+          url
+          updatedAt
+          languages(first:5) {
+          edges  {
+              node {
+                name
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}`;
+
+async function fetchGithubGraphqlApi(query, variables) {
```

Comment:
Instead of `fetchGithubGraphqlApi` and `fetchGithub`, why not `github.graphQuery` and `github.restQuery`? Both of these use `fetch` underneath I believe.

Reply:
@kilemensi Since the `fetchGithubApi` is only used once, we can transform it to a GraphQL query instead of having both implementations.